doc/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ if(${LATEXMK} STREQUAL LATEXMK-NOTFOUND)
 endif()
 
 if(NOT DEFINED SPHINXOPTS)
-  set(SPHINXOPTS -q)
+  set(SPHINXOPTS -j auto -q)
```

Comment:
I tried this but there were some issues with the generated content. Are you sure this generates the very same output?

Reply:
I pulled @mbolivar's patch and generated content locally both with and without the ``-j auto`` on the sphinxbuild command.  There are differences in 10 generated html files, but as I look at the html diffs, the differences are curious but not consequential -- the generated output looks the same (or very close) in the web browser. So, the earlier issues with ``-j auto`` identified in other PRs/Issues didn't show up.  This is encouraging.

Reply:
Same story here. The build produces slightly different HTML but I don't observe differences in the browser. I can drop this part of the series if you prefer.

Reply:
Here's the sequential --> parallel diff.

I took a look at the files in a browser (including a board where the images were different, and the bluetooth page, which had different sidebar HTML) and didn't see any behavioral or visual differences.

Reply:
> For completion @mbolivar, I would be grateful if you could run this on Windows and verify that you also get the same output before setting it to default

I have done this and looked at the results in the same files I checked on Linux (bluetooth.html, the sensor API, Nucleo F207ZG) and they seem fine on Windows as well.